harness/tests/requirements/requirements-harness.txt

```diff
@@ -15,3 +15,4 @@ attrdict3
 moto
 # Pydantic V2 has changes that break existing tests
 pydantic<2
+pandas
```

Comment:
Do we want this to be in `requirements-harness` or should this just be a pip install in `startup-hook.sh` ?